core/src/main/java/org/mule/runtime/core/internal/policy/AbstractCompositePolicy.java

```diff
@@ -13,6 +13,7 @@
 import org.mule.runtime.api.exception.MuleException;
 import org.mule.runtime.core.api.event.CoreEvent;
 import org.mule.runtime.core.api.policy.Policy;
+import org.mule.runtime.core.api.policy.PolicyNotificationHelper;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.mule.runtime.core.api.policy.PolicyNotificationHelper'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7760&issues=AWnGD5CGotIXnmsUaS7j&open=AWnGD5CGotIXnmsUaS7j)

[ISSUE_KEY]: AWnGD5CGotIXnmsUaS7j